learntools/nlp/ex1.py

```diff
@@ -0,0 +1,162 @@
+                            4.488888888888889,
+                            4.238095238095238,
+                            4.166666666666667,
+                            4.666666666666667,
+                            4.142857142857143,
+                            5.0])
+
+        means = np.array([sum(item)/len(item) for _, item in item_ratings.items()])
+        means.sort()
+        correct.sort()
+        assert len(means) == len(correct), f"Please add items to item_ratings. You should have {len(correct)} items."
+        assert np.allclose(means, correct)
+
+class BestReviewedItems(EqualityCheckProblem):
+    _var = "best_items"
+    _hint = ("Loop through each item in item_ratings and calculate the mean, "
+             "the sum of the ratings divided by the number of ratings. This is easiest "
+             "using a dictionary comprehension. Then use the `sorted` function to sort "
+             "the dictionary keys based on the dictionary values.")
+    _solution = CS(textwrap.dedent("""
+    mean_ratings = {item: sum(ratings)/len(ratings) for item, ratings in item_ratings.items()}
+    best_items = sorted(mean_ratings, key=mean_ratings.get, reverse=True)
+    """))
+    _expected = ['artichoke salad',
+                'fettuccini alfredo',
+                'turkey breast',
+                'corned beef',
+                'reuben',
+                'pastrami',
+                'chicken salad',
+                'purista',
+                'prosciutto',
+                'chicken pesto',
+                'chicken spinach salad',
+                'grilled veggie',
+                'gnocchi',
+                'lasagna',
+                'cheesesteak',
+                'pizzas',
+                'pasta',
+                'mac and cheese',
+                'calzone',
+                'cannoli',
+                'pizza',
+                'tiramisu',
+                'ziti',
+                'chicken parmigiana',
+                'salami',
+                'italian sausage',
+                'roast beef',
+                'portobello',
+                'meatball',
+                'garlic bread',
+                'italian beef',
+                'tuna salad',
+                'eggplant',
+                'italian combo',
+                'spaghetti',
+                'turkey sandwich',
+                'chicken cutlet']
+
+class CountImportanceQuestion(ThoughtExperiment):
+    _solution = """
+    The less data you have for any specific item, the less you can trust that the average rating is the "real" sentiment of the customers. This is fairly common sense. If more people tell you the same thing, you're more likely to believe it. It's also mathematically sound. As the number of data points increases, the error on the mean decreases as 1 / sqrt(n).
+    """
+
+qvars = bind_exercises(globals(), [
+    SingleReviewMatch,
+    MatchAllDataset,
+    BestReviewedItems,
+    CountImportanceQuestion
+    ],
+    tutorial_id=262,
```

Comment:
@mcleonard You should remove this line, and then merge master into this branch... since we no longer specify tutorial_id's in learntools

Reply:
Done!